setup.py

```diff
@@ -58,7 +58,7 @@
         "networkx >= 2.5, ==2.*",
         "packaging >=19.2",
         "pandas >=1.0.0, ==1.*",
-        "phylo-treetime ==0.8.*",
+        "phylo-treetime ==0.9.*",
```

Comment:
We may want to start off with the latest bug fix version of treetime here? Why not say `>=0.9.2, ==0.9.*`?